docs/evm/evm-testing.md

```diff
@@ -0,0 +1,55 @@
+---
+id: evm-testing
+title: Testing EVM contracts
+sidebar_label: Testing
```

Comment:
Perhaps the sidebar could say "Testing contracts", as it wasn't clear to me what kind of testing this page would be about.